python/tests/unittest/test_java_app.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 Alibaba Group Holding Limited. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import logging
+import os
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import pytest
+
+from graphscope import JavaApp
+
+DEFAULT_GS_CONFIG_FILE = ".gs_conf.yaml"
+
+logger = logging.getLogger("graphscope")
+
+
+@pytest.fixture(scope="module")
+def not_exist_jar():
+    path = os.path.join("not_exist_dir", "not_exist.jar")
+    return path
+
+
+@pytest.fixture(scope="module")
+def not_jar_file():
+    path = os.path.join("/tmp", "test.txt")
+    Path(path).touch()
+    yield path
+    os.remove(path)
```

Comment:
You could load an arbitrary file (e.g., `xxxx.csv`) in gstest instead of creating a new file.